tests/foreman/ui/test_package.py

```diff
@@ -233,18 +233,27 @@ def test_positive_rh_repo_search_and_check_file_list(
     with session:
         session.organization.select(org_name=module_org.name)
         assert session.package.search(
-            'name = {0}'.format('facter'),
+            'name = {0}'.format('puppet-agent'),
             repository=module_rh_repo.name,
-        )[0]['RPM'].startswith('facter')
+        )[0]['RPM'].startswith('puppet-agent')
         assert session.package.search(
             'name = {0}'.format('katello-host-tools'),
             repository=module_rh_repo.name
         )[0]['RPM'].startswith('katello-host-tools')
         package_details = session.package.read(
-            'katello-agent', repository=module_rh_repo.name)
+            'tracer-common', repository=module_rh_repo.name)
         assert {
-            '/usr/lib/gofer/plugins/katelloplugin.py',
-            '/usr/lib/yum-plugins',
-            '/usr/lib/yum-plugins/package_upload.py',
-            '/usr/sbin/katello-package-upload',
+            '/etc/bash_completion.d/tracer',
+            '/usr/share/doc/tracer-common-0.7.1',
+            '/usr/share/doc/tracer-common-0.7.1/README.md',
+            '/usr/share/licenses/tracer-common-0.7.1',
+            '/usr/share/licenses/tracer-common-0.7.1/LICENSE',
```

Comment:
Can you remove these lines? Presumably `tracer-common` package will be updated some day, and then test will start failing due to versioned files not being present in package.

This assertion already allows for package files to contain more items than we have hardcoded in test.